content/ytt/docs/latest/faq.md

```diff
@@ -101,6 +97,23 @@ The subset matcher does not directly support regex patterns. Instead, a custom m
 
 ---
 
+## When should I include a space in my ytt comment? Does it matter? Is it `#@load` or `#@ load`? `#@overlay/match` or `#@ overlay/match`
+The space is subtly meaningful, and directly, load needs a space, while overlay/match does not – but why?
+
+ytt wraps two concepts in its comment syntax:
+1. Annotations on a node
+1. Directives for ytt
+
+Annotations do not have a space, and they refer to a given node in the tree. These comments attach metadata to the annotated node, which can be used during templating.\
```

Comment:
what are the `\` at the end of lines?

Reply:
The `\` forces the next line of text to begin on a new line.

Reply:
i see. this made me go look at the rendered source a bit. i think it looks pretty confusing for sentences to just start start on a new line even though they seem to be part of the paragraph. lets remove \s.